examples/minimal/packages/contracts/.solhint.json

```diff
@@ -1,11 +1,13 @@
 {
   "extends": "solhint:recommended",
+  "plugins": ["mud"],
   "rules": {
     "compiler-version": ["error", ">=0.8.0"],
     "avoid-low-level-calls": "off",
     "no-inline-assembly": "off",
     "func-visibility": ["warn", { "ignoreConstructors": true }],
     "no-empty-blocks": "off",
-    "no-complex-fallback": "off"
+    "no-complex-fallback": "off",
+    "mud/no-msg-sender": "error"
```

Comment:
any way to make this the default if you enable/use the plugin?

Reply:
a mud extension with defaults, that would be another package in a followup PR